tests/validate_inventory/suites/vendor.yml

```diff
@@ -0,0 +1,26 @@
+tags: validate_vendor
+env:
+  ANSIBLE_ROLES_PATH: roles:tests/validate_inventory/roles_for_vaildating_role_fetching
+template_file: test_inv.yml.j2
+
+tests:
+  - test_name: valid_vendor
+    expected: 0
+
+  - test_name: invalid_test_vendor
+    expected: 2
+    template:
+      vendor: test_custom_vendor
+
+  - test_name: invalid_test_custom_vendor_doesnt_exist
+    expected: 2
+    template:
+      vendor: test_custom_vendor_doesnt_exist
+      allow_custom_vendor: true
+
+  - test_name: valid_test_custom_vendor
+    expected: 0
+    template:
+      vendor: test_custom_vendor
+      allow_custom_vendor: true
+  
```

Comment:
github is complaining about a new line but It looks like there is one. Spacing perhaps?